Makefile

```diff
@@ -3,6 +3,12 @@
 REACT_APP_CMD := EXTEND_ESLINT=true npx react-scripts
 LINTED_FILES := ./src/**/*.ts{,x} ./src/*.ts{,x}
 
+ifeq (,$(findstring w,$(MAKEFLAGS)))
```

Comment:
Can we flip this condition (so that first branch is watchmode) or tag the watchmode branch [it's not apparently obvious which is which]?

Also: could you please add explicit default values (that'd be `CI:=` and so on so on...) and a comment on `CI=false` (that's it's a `react-scripts` way of doing non-interactive, whereas ' ' makes it interactive). One more suggestion: how about setting the `CI` to `true` on non-interactive runs? I don't remember why it was set to `false` in the first place, but from my tests looks like it'll work ;d